frontend/public/style/_common.scss

```diff
@@ -97,12 +97,18 @@
   color: $red;
 }
 
-.co-section-title {
+.co-section-heading {
   font-size: 20px;
   font-weight: 600;
   margin: 0 0 20px 0;
 }
 
+.co-section-title {
+  font-size: 40px;
```

Comment:
Should be `20px` to maintain consistency with existing rule? If so, this could be nested with `.co-section-heading`?